src/pages/EntityDetails/_entity-details.scss

```diff
@@ -294,45 +289,95 @@
     }
   }
 
-  .entity-details__units.has-checkbox {
-    th,
-    td {
-      &:nth-child(1) {
-        width: 4%;
-      }
+  .entity-details__units {
+    &:not(.has-checkbox) {
+      th,
+      td {
+        // Agent.
+        &:nth-child(3) {
+          width: 10%;
+        }
 
-      &:nth-child(2) {
-        width: 20%;
-      }
+        // Machine.
+        &:nth-child(4) {
+          width: 15%;
+        }
 
-      &:nth-child(3) {
-        width: 15%;
-      }
+        @include mobile-and-medium {
+          // Agent.
+          &:nth-child(3) {
+            width: 15%;
+          }
+
+          // Machine.
+          &:nth-child(4),
+            // Message
+            &:nth-child(7),
+            // Ports.
+            &:nth-child(6) {
+            display: none;
+          }
+        }
 
-      &:nth-child(4) {
-        width: 10%;
+        @include mobile {
+          // Agent.
+          &:nth-child(4),
```

Comment:
I am not sure, but shouldn't it be a 3 instead of 4 for the Agent column?

Reply:
Thanks, it seems this column is already hidden in the `mobile-and-medium` query so I've removed this line.